.github/actions/compliance/action.yml

```diff
@@ -72,7 +74,6 @@ runs:
     - name: Generate duvet report
       working-directory: ${{ inputs.dir }}
       run: |
-        ./ci/compliance/extract.sh
```

Comment:
Does this happen somewhere else?

Reply:
Nah, this is the only place the scripts are executed.